src/components/header.component.vue

```diff
@@ -0,0 +1,395 @@
+              </div>
+            </h3>
+          </div>
+          <div class="PopularSearchKeywordList__Row">
+            <div class="PopularSearchKeywordList__Column">
+              <div
+                class="CandidateItem__Container"
+                v-for="(Item, index) in DummyData.popular.slice(-5)"
+                :key="DummyData.popular[index]"
+              >
+                <span class="CadidateItem__Number" v-if="index < 5">{{
+                  index + 1
+                }}</span>
+                <span class="CandidateItem__Text" v-if="index < 5">{{
+                  Item
+                }}</span>
+              </div>
+            </div>
+            <div class="PopularSearchKeywordList__Column">
+              <div
+                class="CandidateItem__Container"
+                v-for="(Item, index) in DummyData.popular.slice(5)"
+                :key="DummyData.popular[index]"
+              >
+                <span class="CadidateItem__Number">{{ index + 6 }}</span>
+                <span class="CandidateItem__Text">{{ Item }}</span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Header',
+  data: () => {
+    return {
+      isShowCategory: false,
+      isShowCartText: true,
+      DummyData: {
+        recommended: [
+          '오늘의 특가',
+          '아이패드0원',
+          'TOP20',
+          '1+1클래스',
+          '여름 취미 추천',
+          '감사제 특가',
+          '10만원할인',
+        ],
+        popular: [
+          '아이패드',
+          '일러스트',
+          '이모티콘',
+          '드로잉',
+          '포토샵',
+          '뜨개질',
+          '레진',
+          '주식',
+          '포토크리에이트',
+          '가전주부 PICK',
+        ],
+      },
+    }
+  },
+  computed: {
+    SearchInputHandler: function() {
+      if (this.isShowCategory == true) {
+        return {
+          width: '676px',
+          margin: 'auto',
+          'align-items': 'center',
+        }
+      } else {
+        return {
+          width: '420px',
+        }
+      }
+    },
+    SearchContainerHandler: function() {
+      if (this.isShowCategory == true) {
+        document.body.style.cssText = `overflow: hidden;`
+        return {
+          display: 'flex',
+          'flex-direction': 'row',
+        }
+      } else {
+        return {
+          position: 'realtive',
+          flex: '1 1 0%',
+        }
+      }
+    },
+  },
+}
+</script>
+
+<style lang="scss">
+input::-ms-clear,
+input::-ms-reveal {
+  display: none;
+  width: 0;
+  height: 0;
+}
+
+input::-webkit-search-decoration,
+input::-webkit-search-cancel-button,
+input::-webkit-search-results-button,
+input::-webkit-search-results-decoration {
+  display: none;
+}
+
+.HeaderWrapper {
+  width: 100%;
+  height: 5.375rem;
+  .HeaderBox {
+    display: flex;
+    padding: 20px 0px;
+    font-size: 0px;
+    justify-content: center;
+    height: 48px;
+    width: 100%;
```

Comment:
css도 비슷한 속성들이 있습니다.
justify-content 속성은 flex가 아니면 아무런 효과도 없습니다. 따라서 두 속성은 모아주는게 좋겠죠

```suggestion
    display: flex;
    justify-content: center;
    width: 100%;
    height: 48px;
    padding: 20px 0px;
    font-size: 0px;
```